### ISSUE TITLE
Connecting remote resets video

### ISSUE THREAD

**Issue by TheCyberConnect** [yuliskov/SmartTubeNext#2172]
Hello I love the app and thank you for all the work you've done. 

I use a Bluetooth keyboard that goes to sleep when idle to preserve battery. When the keyboard goes to sleep or it reconnects from sleep, the video either resets or goes to a black screen for a second before resuming. 

This doesn't occur with any other streaming service or regular YouTube.

**Comment by antonLavrikConstrafor** [comment]
Yeah, it's realy annoying. I switch beetween devices on my keyboard. When i pause video and switch to another device video start again!
